CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+# Contributing
+
+Everyone is welcome to contribute to GoDaddy's Open Source Software.
+Contributing doesn’t just mean submitting pull requests. To get involved,
+you can report or triage bugs, and participate in discussions on the
+evolution of each project.
+
+No matter how you want to get involved, we ask that you first learn what’s
+expected of anyone who participates in the project by reading the Contribution
+Guidelines and our [Code of Conduct][coc].
+
+**Please Note:** GitHub is for bug reports and contributions primarily -
+if you have a support question head over to [GoDaddy's Open Source
+Software Slack channel][slack]. You can request an invite
```

Comment:
Does this really need to have 2 separate links for our slack server? Isn't just the invite enough? Seems a bit odd to have 2 different links for it.

Reply:
Repetition in on-boarding documentation has paid dividends in a few past experiences.